00--Ready-Player-One/README.md

```diff
@@ -38,19 +95,12 @@
     * [ ] [Atom](http://atom.io)
     * [ ] [Dash](http://kapeli.com/dash)
     * [ ] [Slack](http://www.slack.com)
-* [ ] **Send all my GitHub to you...**
- * [ ] [Complete the "Hello World" GitHub guide](https://guides.github.com/activities/hello-world/)
- * [ ] Add a link to your new GitHub repo as a comment on your *WIP Issue*
- * [ ] [Complete the "Forking Projects" GitHub guide](https://guides.github.com/activities/forking/)
- * [ ] Add a link to your `Fork-and-Spoon` repo as a comment on your *WIP Issue*
+    * [ ] [SourceTree](https://www.sourcetreeapp.com/)
+* [ ] **Prezto Change-oh!**
+ * [ ] Find and _fork_ the Prezto project on GitHub
+ * [ ] Clone _your fork_ of Prezto into your home directory
+ * [ ] Follow the installation instructions in the `README.md` file
+* [ ] **Reflection**
+ * [ ] Answered both questions in short essay form
+ * [ ] Added both answers as comments on your WIP Issue
```

Comment:
Maybe list the two questions again? 

> * Added a comment for "What is the most challenging thing..."
> * Added a comment for "What did you HATE..."

Or whatever makes sense. These checkboxes should be action items that correspond (roughly) to the "deliverables" from the assignment.